src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -55,6 +55,8 @@ Running a node allows you to trustlessly and privately use Ethereum while suppor
 
 Running your own node enables you to use Ethereum in a truly private, self-sufficient and trustless manner. You don't need to trust the network because you can verify the data yourself with your client. "Don't trust, verify" is a popular blockchain mantra.
 
+- You can run the programs using your node on the same computer. This allows you to connect to your node using IPC or possibly through rewriting the node to load your program as a plugin bringing low latency which is a requirement when you to replace your transactions as fast as possible (aka frontrunning).
```

Comment:
Not sure I fully understand this. Is it a point about being able to investigate the blockchain locally on your computer? Or about frontrunning? If both, I would consider breaking into two, and placing the frontrunning information bottom on the list as it's a more advanced topic.

Reply:
@wackerow this is only about frontrunning. I’m meaning, the ability to detect and replace transactions as fast as possible.